mlir/lib/Tools/mlir-opt/MlirOptMain.cpp

```diff
@@ -281,7 +282,8 @@ static LogicalResult doVerifyRoundTrip(Operation *op,
     roundtripModule =
         parseSourceString<Operation *>(ostream.str(), parseConfig);
     if (!roundtripModule) {
-      op->emitOpError() << "failed to parse bytecode back, cannot verify round-trip.\n";
+      op->emitOpError()
```

Comment:
whitespace changes? Did the file need reformatting?

Reply:
> whitespace changes? Did the file need reformatting?

Weird, I think my clang-format is acting out somehow here. Wait, the reformatting looks like needed since the original code has 88 characters in the line where llvm coding style says it should be ["within 80 columns" ](https://releases.llvm.org/10.0.0/docs/CodingStandards.html#source-code-width) 🤔.